server/controllers/index.js

```diff
@@ -72,72 +94,86 @@ router.get('/api/start', auth, (req, res) => {
         res.sendStatus(200);
     } catch (err) {
         console.log(err);
-        res.status(400).json({message: err.message});
+        res.status(400).json({ message: err.message });
     }
 });
 
 /* End the game */
 router.get('/api/end', auth, (req, res) => {
     const { roomId } = req.session;
     try {
-        const gameState = Room.getById(req.session.roomId).gameState;
-        gameState.endGame();
-        Room.deleteById(roomId);
-        closeRoom(roomId);
-        res.sendStatus(200);
+        const gameState = Room.getById(roomId).gameState;        
+        gameState.getPlayers().forEach(player => {
```

Comment:
Is this a sequelize thing? I'm a bit confused

Reply:
No that was there before, Room is a file you defined I think haha i'm getting a bit confused too

Reply:
Oh I wasn't talking about Room I meant the stuff under this line sorry

Reply:
it was a sequelize thing yeah, but it's now different and hopefully clearer